scrapers_next/in/committees.py

```diff
@@ -0,0 +1,81 @@
+from spatula import URL, HtmlListPage, SkipItem
```

Comment:
Don't import `HtmlListPage`, import `HtmlPage` instead.

Reply:
If I import HtmlPage only then I get this error
ERROR:spatula:found no list pages in module scrapers_next.in.committees

Reply:
Then that's fine, import both, and just convert only the `CommitteeDetail` class from `HtmlListPage` to `HtmlPage` object.

This ended up being a special case where you used spatula's Html page scrapers to process a JSON response, when we typically use the `JsonListPage` and `JsonPage` spatula objects for that. It worked fine, so we can move forward with it this way, but in the future:
- Use either the `HtmlListPage` or `JsonListPage` class (depending on whether you're processing HTML or JSON) and use the `process_item()` method to iterate through list of items (and use a selector to define the list of items). Then, pass to a `HtmlPage` or `JsonPage` class for scraping the details from any additional page that needs to be called.